src/Catalog/Helpers/AuditRecordHelpers.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Packaging.Core;
+using NuGet.Services.Metadata.Catalog.Persistence;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    /// <summary>
+    /// Helper methods for accessing and parsing audit records.
+    /// </summary>
+    public static class AuditRecordHelpers
+    {
+        /// <summary>
+        /// Represents the information in the audit entry for a deletion.
+        /// </summary>
+        public class DeletionAuditEntry
+        {
+            /// <summary>
+            /// Over time, we have had three "deleted" file names.
```

Comment:
(will probably take out the explicit reference to a number so we don't have to keep updating this number)